### PR TITLE
SNOW-852430 Add external browser timeout docs

### PR DESCRIPTION
**Pull request by sfc-gh-pfus**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -306,6 +306,10 @@ If you are using a different method for authentication, see the examples below:
 
   * `{login_name_for_IdP}` is your login name for your IdP.
 
+  You can also override the default timeout after which external browser authentication is marked as failed.
```
Comment (sfc-gh-pbulawa) [inline README.md:309]: I would remove `also` word.

File: README.md [hunk 2]
```diff
@@ -306,6 +306,10 @@ If you are using a different method for authentication, see the examples below:
...
+  This prevents the situation in which client decides to quit a browser tab.
```
Comment (sfc-gh-pbulawa) [inline README.md:310]: How about `The timeout prevents the infinite hang when the user does not provide the login details, e.g. when closing the browser tab`?

File: README.md (hunk 1 above)
Comment (sfc-gh-pfus) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (sfc-gh-pfus) [reply README.md]: Wonderful!

**Review by sfc-gh-pbulawa — APPROVED** [review]

**Pull request merged**